style.css

```diff
@@ -15,9 +15,9 @@ Text Domain: planet4-child-theme-israel
 @import url('https://fonts.googleapis.com/css2?family=Assistant:wght@400;700&display=swap');
```

Comment:
If you are no longer using Assistant, you don't need to fetch the font here. 

You may also want to fetch Arial instead, since otherwise systems that don't have Arial would fallback to their own default font.